src/crimson/net/SocketConnection.h

```diff
@@ -41,6 +41,10 @@ class SocketConnection : public Connection {
   };
   side_t side = side_t::none;
   uint16_t socket_port = 0;
+  void set_socket_port(uint16_t port, side_t _side) {
```

Comment:
I think it's ok, `set_port()` IMO is confusing because it is not setting the bound port of the local msgr, nor the bound port of the peer msgr we connect to.

Reply:
but "socket" here stands for transport layer, i cannot tell if socket_port means the bound port or a ephemeral port, just by looking at its name.

Reply:
Yup, `set_ephemeral_port()` might be better, changed.

Reply:
i am fine if we want to make this explicit. but i think there is no ambiguity here. as it only makes sense to set the bind port of messenger. and when it comes to connection, the remote port (bind port) is determined before the connection is established. so if there is an API allows one to change the port of a connection, that port should be an ephemeral / high port.